docs/podman.1.md

```diff
@@ -72,6 +72,9 @@ Default state dir is configured in /etc/containers/storage.conf.
 
 Name of the OCI runtime as specified in libpod.conf or absolute path to the OCI compatible binary used to run containers.
 
+**--network-cmd-path**=**path**
+Path to the slirp4netns binary to use for setting up a slirp network.  If "" is used then the binary is looked up using the $PATH environment variable.
+
```

Comment:
I'd use the wording from libpod.conf.5.md here too, making it just a bit more generic.

Reply:
thanks, fixed